tweaks/steam.yaml

```diff
@@ -1107,8 +1165,8 @@
 "207490": # Rayman Origins
   compat_tool: proton_63
 
-"1158450": # GALAXIUM
-  compat_tool: proton_63
+"1067540": # RÖki
```

Comment:
This is actually a duplicate.

Reply:
GALAXIUM is wrongly sorted. I move it up to line 594.

Reply:
I mean Roki, there is another instance of it later in the file.